resources/views/forum/rss.blade.php

```diff
@@ -15,8 +15,8 @@
 <rss version="2.0" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:atom="http://www.w3.org/2005/Atom">
 	<channel>
 		<copyright>Copyright {{date('Y')}} C.S.R. Delft</copyright>
-		<pubDate>{{rfc2822(null)}}</pubDate>
-		<lastBuildDate>{{rfc2822(null)}}</lastBuildDate>
+		<pubDate>{{rfc2822(date_default_timezone_get())}}</pubDate>
```

Comment:
Voelt als een hack

Reply:
Het werkt.
"Facts don't care about your feelings."

Reply:
Maar een alternatief is dat op magische wijze de datum van de laatste forumpost bij 'pubDate' wordt geplaatst. Geen idee hoe dat moet...

Reply:
pubdate moet de laaste forumpost zijn
lastbuilddate moet de laatste edit zijn